### ISSUE TITLE
[dashboard] Replace context URLs and labels

### ISSUE THREAD

**Issue by gtsiolis** [gitpod-io/gitpod#3594]
### Problem to solve

The design specs in #3426 for the new Workspaces list contained some structure and labels for the Context URL. For the time being, we're surfacing the context URl which overflows on smaller viewports and includes some duplicate information with the _Pending Changes_ column on the right.

### Proposal

More particularly, on the second column, a) the context URL should not include the branch name, b) the context URL should surface the commit message, and c) the subtitle should be more semantic like _Pull Request 324_,.

**Comment by apolopena** [comment]
Would the branch information then go into the name column? I was not seeing the branch information anywhere but the context column in the V2 [design](https://www.figma.com/file/0saJp50UikQJMOB36txeZi/Workspaces-%233426?node-id=0%3A1)
The icons are helpful in the V1 design, maybe they can stay in V2 somehow?
![image](https://user-images.githubusercontent.com/3060702/112668905-430bb000-8e1c-11eb-81ad-d3e5827d3ccd.png)

**Comment by svenefftinge** [comment]
I'm not sure I understand what's being discussed here, but #3627 might improve the situation by making the context url an actual link. With that you can go to the context or copy the url.

**Comment by apolopena** [comment]
@svenefftinge 
Yes #3627 does help.  I was discussing placement of the data in regard to the context URL and not loosing any visual cues in the redesign that would inform the user as to what data they are actually looking at.

**Regarding the redesign**
The new design does away with the icons and has less space to display the data but being compact is good thing so:

**For Clarity**
- Could the full urls in a cell be displayed in a tool tip when rolling over a link? 

  - _Without the icons and with the urls being truncatated, it can be unclear what the urls are for._

**For Consistency and functionality**
- Cells in the Name column have two parts (workspace URL, repo URL), each are links. This is good UI.
- Context cells have two parts as well (branch URL and I am not sure what the lower link is for), however the top part (branch URL) is not a link.
  - _Could both parts in a Context 'cell' be a link?_

**Comment by gtsiolis** [comment]
What do you think of this feature request, @jldec?
